Sources/Host/Darwin/X86_64/MachX86_64.cpp

```diff
@@ -22,8 +22,6 @@
 #include <sys/uio.h>
 #include <sys/user.h>
 
-#define super ds2::Host::POSIX::Mach
```

Comment:
Yeah I don't know how this was committed in the first place 😆 

Should I submit this commit right away?